alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-input-v2.component.ts

```diff
@@ -535,14 +542,16 @@ export class DecisionInputV2Component implements OnInit, OnDestroy {
     this.conditionUpdates = $event;
   }
 
-  onChangeDecisionOutcome($event: any) {
-    if ($event.code === 'APPR' || $event.code === 'APPA') {
-      this.showComponents = true;
-      this.form.controls.isSubjectToConditions.enable();
-      this.form.patchValue({
-        isSubjectToConditions: null,
-      });
-    } else {
+  onChangeDecisionOutcome(selectedOutcome: DecisionOutcomeCodeDto) {
+    if (['APPR', 'APPA', 'RESC'].includes(selectedOutcome.code)) {
+      if (this.form.controls.isSubjectToConditions.disabled) {
```

Comment:
I don't want it to go into the else statement if the selected value was APPR, APPA, or RESC. If I combine them into a single if, it will then check the else if.